test/plots/piePlotTests.ts

```diff
@@ -412,14 +348,43 @@ describe("Plots", () => {
         svg.remove();
       });
 
-      it("skips invalid Datsets", () => {
+      it("skips invalid Datasets", () => {
         let allSectors = piePlot.selections([new Plottable.Dataset([])]);
         assert.strictEqual(allSectors.size(), 0, "no sectors retrieved");
 
         svg.remove();
       });
 
-      it("retrieves entities under a point with entitiesAt()", () => {
+    });
+
+    describe("entities", () => {
+      let svg: d3.Selection<void>;
+      let dataset: Plottable.Dataset;
+      let simpleData = [{value: 5}, {value: 15}];
+      let piePlot: Plottable.Plots.Pie;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(500, 500);
+        dataset = new Plottable.Dataset();
+        piePlot = new Plottable.Plots.Pie();
+        dataset.data(simpleData);
+        piePlot.addDataset(dataset);
+        piePlot.sectorValue((d) => d.value);
+        piePlot.renderTo(svg);
+      });
+
+      it("returns both a fill path and a stroke path in each Entity's selection", () => {
+        let entities = piePlot.entities();
+        assert.lengthOf(entities, simpleData.length, "returned one Entity per datum");
+        entities.forEach((entity) => {
+          assert.strictEqual(entity.selection.size(), 2, "each entity selection has 2 paths");
+          assert.strictEqual(entity.selection.filter(".fill").size(), 1, "each entity selection has 1 fill path");
+          assert.strictEqual(entity.selection.filter(".outline").size(), 1, "each entity selection has 1 stroke path");
```

Comment:
If one of these test messages fails, there is no way to tell which of the 2 entities did not have the required attributes. Let's introduce an index in the assert message. Something like 

```
"entity " + i + " selection has [...]"
```

Or the shorthand
```typescript
`entity ${i} selection has [...]`
```